tutorials/experiments/prompt-optimization.ipynb

```diff
@@ -0,0 +1,2121 @@
+{
```

Comment:
<p>Don't need "finally" here. Might just educate the user that experiment runs are traced in Phoenix.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/6472/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>